trogon/widgets/parameter_controls.py

```diff
@@ -406,23 +406,27 @@ def _make_command_form_control_label(
         is_required: bool,
         multiple: bool,
     ) -> Text:
-        if isinstance(name, str):
-            text = Text.from_markup(
-                f"{name}[dim]{' multiple' if multiple else ''} {type.name}[/] {' [b red]*[/]required' if is_required else ''}"
-            )
-        else:
-            names = Text(" / ", style="dim").join([Text(n) for n in name])
-            text = Text.from_markup(
-                f"{names}[dim]{' multiple' if multiple else ''} {type.name}[/] {' [b red]*[/]required' if is_required else ''}"
```

Comment:
The `{name}` part here is a bug as `Text.plain` is used when a `Text` object is referenced in an f-string, so we lose the `dim` style in the rendered result.

However, it looks better without the `dim`, so I did not include the `dim` in this PR.